packages/sdk/src/utils/errors/errorMapping.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
This also can be removed. It's for the v1 smart contracts.

Reply:
removed [c36bf33](https://github.com/Voltz-Protocol/v2-off-chain-monorepo/pull/10/commits/c36bf330f7daf2dab92b10c2d0c1ef4b5e4e1cc0)